change/@fluentui-react-native-tester-6ddb0f57-0907-4108-acfa-76fa6526c71b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add Experimental Menu Button",
```

Comment:
nit: I'd probably change this comment to better reflect the changes made (for example: Add additional E2E testing and info to the README)